.github/workflows/install_deps/action.yml

```diff
@@ -63,6 +73,13 @@ runs:
         pip install -r requirements-ci.txt --upgrade
         pip install -r requirements-dev.txt --upgrade
 
+    - name: Install numpy 2.0 version
+      shell: bash
+      if: inputs.install_numpy_2 == true
+      env:
+        NUMPY2_VERSION: ${{ inputs.numpy_2_version != '' && format('~={0}', inputs.numpy_2_version) || '' }}
```

Comment:
What does this string evaluate to? Is it `=2.0.0`  when set?

Reply:
If set, it would be `numpy~=2.0.0`

Reply:
Yep, `~=2.0.0` if `inputs.numpy_2_version` is `2.0.0` (default value), otherwise an empty string if `inputs.numpy_2_version` is also an empty string